simple_history/templatetags/simple_history_admin_list.py

```diff
@@ -1,8 +0,0 @@
-from django import template
-
-register = template.Library()
-
-
-@register.inclusion_tag("simple_history/_object_history_list.html", takes_context=True)
-def display_list(context):
```

Comment:
Rather than removing it outright, maybe it should raise a deprecation warning?

Reply:
Hm yeah, it didn't occur to me that some users might be using this for overriding parts of the template(s) - if that's what you were thinking about - so I agree that we should deprecate it first.

Reply:
I don't have a lot of experience deprecating things in Python/Django libraries, so let me know if you're familiar with any technique that would be preferred over what I added in 8513231192aba4e52d51bbe708dc716f162e6a3a 🙂 Also, any opinions on whether 3.8 sounds like a good version to remove the template tag?

Reply:
I think if our project had a regular release cadence we could have a formal deprecation policy. I think what you'd laid out works perfectly well!